code/modules/fish/fish_items.dm

```diff
@@ -152,15 +152,15 @@
 	name = "shark teeth"
 	desc = "A number of teeth, supposedly from a shark."
 	icon = 'icons/obj/fish_items.dmi'
-	icon_state = "teeth"
 	force = 2.0
 	throwforce = 5.0
 	materials = list()
 
-/obj/item/shard/shark_teeth/Initialize(mapload)
+/obj/item/shard/shark_teeth/initial_icon_state()
 	. = ..()
```

Comment:
```suggestion
```
You'd not want this as you want to fully override the previous behaviour instead (Except for the caltrop component but see the other comments for that)